trunk/SUAVE/Methods/Geometry/Two_Dimensional/Cross_Section/Airfoil/compute_airfoil_polars.py

```diff
@@ -149,7 +148,30 @@ def compute_airfoil_polars(a_geo,a_polar):
             # Pack this loop
             CL[i,j,:] = CL_ij
             CD[i,j,:] = CD_ij
-           
+            
+            if use_pre_stall_data == True:
+                # Coefficients in pre-stall regime taken from experimental data:
+                aoa_locs = (AoA_sweep_deg>=airfoil_aoa[0]) * (AoA_sweep_deg<=airfoil_aoa[-1])
+                aoa_in_data = AoA_sweep_deg[aoa_locs]
+                
+                cl_exp = np.zeros_like(aoa_in_data)
+                cd_exp = np.zeros_like(aoa_in_data)
+                for a in range(len(aoa_in_data)): 
```

Comment:
This loop  is vectorizable. Let me know if you have questions about that.

Reply:
done